reference-sheets/FileStructure.md

```diff
@@ -0,0 +1,36 @@
+# Introduction
+This guide explains the structure of a typical robotics project, focusing on how files and directories are organized to maintain a clear codebase. It covers the distribution of library versus robot-specific code and the purpose of each major file. 
+
+All of the source code for a project is in the [src](https://github.com/SciBorgs/SciGuidesRobotBase/tree/main/src) folder. Within that folder, there is the following directory structure:
+
+![](/images/sciguides-robot-base-file-tree-dirs.png)
+
+There are two top level directories under `src`: `main`, for code that runs on the robot, and `test`, for unit tests.
+
+We're going to ignore the `deploy` directory, since it isn't very relevant to us right now. That leaves us with the contents of `main/java` and `test/java` (both of which contain Java files). The robot code in `main/java` is split into a `lib` folder, which contains library code that is not specific to one particular robot, and a `robot` folder, which contains the code to control a particular robot. Classes in the robot folder generally use lib classes.
+
+You may notice that the `test/java` directory actually has the same structure. This is because our `test` folder aims to test code in `main`, and so inside of `test` we mirror the structure of `main`. If we are a file `Foo.java` at the address `main/java/lib/Foo.java`, our test for that file would be at the address `test/java/lib/FooTest.java`.
+
+Now, let's look at a tree that includes the actual files as well as directories:
+
+![](/images/sciguides-robot-base-file-tree.png)
+
+There's a lot here, but we'll go through it piece by piece.
+
+One thing you might notice is that the `lib` folders are very populated. That's because one of the main points of using this template is that it has library code from the SciBorgs, which you can use in your project. You will not be writing code in the `lib` folders, but you will be using the utilities from those folders. But you don't have to worry about that right now, we'll introduce specific library code when it is relevant.
+
+So we're just going to focus on the contents of the `robot` directories. Let's start with `main/java/robot`:
```

Comment:
Section is a bit wordy, "this is a file" can be removed and is obvious but its pedantic but should be quick enough